infrastructure/public-api-config.yaml

```diff
@@ -0,0 +1,53 @@
+plugin: openapi
+specFile: ./public-api.yaml
+
+validation:
+  request: log
+
+resources:
+
+  - method: POST
+    path: /token
+    formParams:
+      code: aaaaaaaaa
+    response:
+      statusCode: 400
+
+  - method: POST
+    path: /token
+    formParams:
+      code: null
+    response:
+      statusCode: 500
+
+  - method: POST
+    path: /token
+    response:
+      statusCode: 201
+
+  - method: POST
+    path: /credential/issue
+    response:
+      statusCode: 200
+
+  - method: POST
+    path: /credential/issue
+    requestHeaders:
+      SessionHeader: Authorization Bearer 123456789
```

Comment:
Same as above - this should be session-id not SessionHeader
Also I think you've got a duplicate with the block above, shouldn't have more than one 200 response